test/cppsuite/test_harness/database_operation.h

```diff
@@ -0,0 +1,269 @@
+/*-
```

Comment:
The functions in this file all come from workload_generator.h. This is their default implementation.